src/api/tokenList/Faucet_info.md

```diff
@@ -0,0 +1,53 @@
+Ether, Tether, TrueUSD, USD Coin, Paxos, Gemini, Dai
```

Comment:
Can you add a bit of format and create a link the file in the main README.md? it's being displayed like this: https://github.com/gnosis/dex-react/blob/token_list_json/src/api/tokenList/Faucet_info.md

For example:
```md
# Faucelet information
## Wrapped Ether
* [https://weth.io](https://weth.io)
* Wrapper of Ether to make it ERC-20 compliant
* **Rinkeby faucet**: [https://faucet.rinkeby.io/](https://faucet.rinkeby.io)
...
```